drivers/driver_env.go

```diff
@@ -0,0 +1,41 @@
+package drivers
+
+import (
+	"crypto/rsa"
+	log "github.com/Sirupsen/logrus"
+	"github.com/rancher/rancher-auth-service/util"
+	"github.com/urfave/cli"
+	"os"
+)
+
+//PrivateKey to be used for jwt creation
+var PrivateKey *rsa.PrivateKey
+
+//PublicKey to be used for jwt decode
+var PublicKey *rsa.PublicKey
+
+var cattleUrl string
+var accessKey string
+var secretKey string
+
+func SetEnv(c *cli.Context) {
+	privateKeyFile := c.GlobalString("rsa-private-key-file")
+	if privateKeyFile != "" {
+		PrivateKey = util.ParsePrivateKey(privateKeyFile)
+	} else {
+		log.Fatal("Please provide rsa-private-key-file, halting")
```

Comment:
You should avoid logging fatal anywhere other than in main.go. FYI, Fatal cause the program to immediately exit with a non-zero exit code. You should instead return an error and handle it in main.go